Multiple testing p-value adjustment for CD diagram

PascalIversen opened the pull request:
Thought this would be the default T.T

PascalIversen committed:
what a miss

JudithBernett reviewed (approved):
aber warum eig holm und nicht benjamini hochberg oder so?

PascalIversen committed:
Change p-value adjustment method to fdr_bh

Benjamini/Hochberg has more power than holm. Arguably, more important than a strict FWER control for our application.
